config

```diff
@@ -1,4 +1,4 @@
 [Config]
 location=location_here
 
-metasmoke_host=http://127.0.0.1:3000
+metasmoke_host=dvsbsbsfb
```

Comment:
yeah.... it was a shoddy commit, I thought I reverted it... on another note, perhaps we should be adding the config file to gitignore